awx/main/management/commands/cleanup_host_metrics.py

```diff
@@ -1,22 +1,24 @@
-from awx.main.models import HostMetric
 from django.core.management.base import BaseCommand
-from django.conf import settings
+from django.utils.translation import gettext_lazy as _
+from awx.main.tasks.host_metrics import HostMetricTask
 
 
 class Command(BaseCommand):
     """
-    Run soft-deleting of HostMetrics
+    Run soft/hard-deleting of HostMetrics
     """
 
-    help = 'Run soft-deleting of HostMetrics'
+    help = 'Run soft/hard-deleting of HostMetrics'
 
     def add_arguments(self, parser):
-        parser.add_argument('--months-ago', type=int, dest='months-ago', action='store', help='Threshold in months for soft-deleting')
+        parser.add_argument('--soft', type=int, nargs='?', default=None, const=0, help='Threshold in months for soft-deleting')
+        parser.add_argument('--hard', type=int, nargs='?', default=None, const=0, help='Threshold in months for hard-deleting')
```

Comment:
It would be good for the help text to say "months ago" because I find this non-obvious.

Reply:
Can you type the whole sentence? 
- "Mark host metrics last automated <x> months ago"
- "Delete host metrics soft deleted <y> months ago"

Reply:
I got an answer that it's not wanted to allow configurable command. It'll be driven by settings only.